code/modules/mapping/map_template.dm

```diff
@@ -152,9 +152,9 @@
 		T = locate(T.x - round(width/2) , T.y - round(height/2) , T.z)
 	if(!T)
 		return
-	if(T.x+width > world.maxx)
+	if((T.x+width) - 1 > world.maxx)
```

Comment:
What's this?

Reply:
fixing the calculations for whether or not attempting to load a map at a specificed coord will go out of bounds.
width is one based not zero based so you need to subtract one from the calculation